pkg/skaffold/build/jib/init.go

```diff
@@ -89,13 +89,30 @@ func validate(path string, enableGradleAnalysis bool) []ArtifactConfig {
 	var builderType PluginType
 	var executable, wrapper, taskName, searchString, consoleFlag string
 	switch {
-	case strings.HasSuffix(path, "pom.xml"):
+	case isPomFile(path):
 		builderType = JibMaven
 		executable = "mvn"
 		wrapper = "mvnw"
 		searchString = "<artifactId>jib-maven-plugin</artifactId>"
 		taskName = "jib:_skaffold-init"
 		consoleFlag = "--batch-mode"
+
+		// Polyglot maven support: support pom files of different extension types
```

Comment:
There's an easier and more robust way: polyglot maven [requires a `.mvn/extensions.xml` file](https://github.com/takari/polyglot-maven#usage).  Polyglot supports a bunch of other languages, including yaml 🙀